packages/admin/cms-admin/src/redirects/RedirectForm.tsx

```diff
@@ -254,6 +254,8 @@ export const RedirectForm = ({ mode, id, linkBlock, scope }: Props): JSX.Element
                                             // eslint-disable-next-line @typescript-eslint/no-explicit-any
                                             validate={validateSource as any}
                                             fullWidth
+                                            placeholder="/example-path"
```

Comment:
I thought it's not necessary because it's an URL. I also thought if we make it translateable, the translators might put disallowed special characters in the placeholder which is confusing.